docs/modules/faqs.html

```diff
@@ -69,8 +69,8 @@ <h1>Module FAQs</h1>
 			<section class="tsd-panel tsd-comment">
 				<div class="tsd-comment tsd-typography">
 					<div class="lead">
-						<a href="#amazon-chime-sdk-for-javascript---frequently-asked-questions" id="amazon-chime-sdk-for-javascript---frequently-asked-questions" style="color: inherit; text-decoration: none;">
-							<h1>Amazon Chime SDK for JavaScript - Frequently Asked Questions</h1>
+						<a href="#frequently-asked-questions" id="frequently-asked-questions" style="color: inherit; text-decoration: none;">
+							<h1>Frequently Asked Questions</h1>
```

Comment:
FYI, it looks like there's some lnline CSS being used here already. You could do something like 
```
<h1 style="line-height: 1.9"
```

And that would likely fix your squashed text problem.